src/Mod/PartDesign/Gui/Utils.cpp

```diff
@@ -171,11 +238,7 @@ PartDesign::Body * makeBody(App::Document *doc)
                              doc->getName(), bodyName.c_str() );
     auto body = dynamic_cast<PartDesign::Body*>(doc->getObject(bodyName.c_str()));
     if(body) {
```

Comment:
Purely stylistic, but the Qt standard would suggest dropping the curly braces:
```suggestion
    if(body)
        makeBodyActive(body, doc);
```

Reply:
Could you point me to the standards? What about the if-else statements with one line each?

Reply:
Here's the Qt standard: https://wiki.qt.io/Coding_Conventions

Keep in mind that this isn't the "official" FreeCAD standard: officially each coder is free to do as they please, and not following this is certainly not a reason to reject a PR. But this is the standard that seems to have the most traction in the current codebase, so it's the one I've been (more or less) adhering to.